master_xml.py

```diff
@@ -124,6 +124,9 @@ def evaluateInfoData(cards):
                             if abilityValue != None:
                                 cards[cardId]['info'][region] = cards[cardId]['info'][region].replace("{" + key + "}", abilityValue)
 
+            cards[cardId]['infoFormatted'][region] = cards[cardId]['info'][region]
```

Comment:
@Zolmeister @Tri125 

Thinking about this, I don't think we should support lots of different formatting methods. It should be up to the developer to work out the formatting. Otherwise we are going to end up with an incredibly bloated data set.

My suggestion would be to rename this new field to `infoRaw` to indicate that nothing has been stripped from the original source. Leave `info` as it is; stripped of all formatting.

What do you think?